apps/charterafrica/src/pages/knowledge/[slug].page.js

```diff
@@ -0,0 +1,102 @@
+import Explainers from "@/charterafrica/components/Explainers";
+import PageInfo from "@/charterafrica/components/PageInfo";
+import ResearchComponent from "@/charterafrica/components/Research";
```

Comment:
```suggestion
import Research from "@/charterafrica/components/Research";
```

Shouldn't we call this Articles/Posts, etc., since it should cover both Research & News?